scripts/eosio_build_centos.sh

```diff
@@ -6,7 +6,7 @@ echo "Disk space total: ${DISK_TOTAL}G"
 echo "Disk space available: ${DISK_AVAIL}G"
 
 ( [[ $NAME == "CentOS Linux" ]] && [[ "$(echo ${VERSION} | sed 's/ .*//g')" < 7 ]] ) && echo " - You must be running Centos 7 or higher to install EOSIO." && exit 1
-
+( [[ $NAME == "CentOS Linux" ]] && [[ "$(echo ${VERSION} | sed 's/ .*//g')" == 8 ]] ) && echo " - We don't support CentOS 8 any more ." && exit 1
```

Comment:
>  \- We don't support CentOS 8 any more .  
  
Should we make this message more specific or helpful? What about this?

> CentOS 8 has been deprecated by the RHEL community and is no longer supported by EOSIO. Please use CentOS 7 or Amazon Linux 2 instead.

Reply:
Good sentence.